src/extremeweatherbench/evaluate.py

```diff
@@ -36,9 +36,8 @@ def evaluate(
         A dictionary mapping event types to lists of xarray Datasets containing the
         evaluation results for each case within the event type.
     """
-
-    base_dir = os.path.dirname(os.path.abspath(__file__))
-    events_file_path = os.path.join(base_dir, "../../assets/data/events.yaml")
+    base_dir = Path(__file__).resolve().parent
+    events_file_path = base_dir / "assets" / "data" / "events.yaml"
```

Comment:
Generally we don't want to hard-code paths like this; Python provides tools to package and ship data with distributions, and we want to leverage that machinery. You want to look up [importlib](https://docs.python.org/3/library/importlib.html) and follow through with how to register datasets in your package via the **pyproject.toml** listings.

You can see a worked example of how this winds up implemented in code [here](https://github.com/brightbandtech/debufr/blob/cb5b910353e20aaa4daa0ffa7afc81fdb2605989/src/debufr/code_flag.py#L163).